lib/voucherify/service/products.rb

```diff
@@ -21,8 +21,8 @@ def update(product)
         @client.put("/products/#{URI.encode(product['id'] || product[:id])}", product.to_json)
       end
 
-      def delete(product_id)
-        @client.delete("/products/#{URI.encode(product_id)}")
+      def delete(product_id, params = {})
+        @client.delete("/products/#{URI.encode(product_id)}", {:force => (!!(params['force'] || params[:force])).to_s})
```

Comment:
I would just pass the params here. If there will be more params in the future to pass to the api endpoint call then this would become hard to maintain.